composer.json

```diff
@@ -14,15 +14,15 @@
     },
     "require": {
         "php": ">=5.5",
-        "zendframework/zend-crypt": "~2.5",
+        "zendframework/zend-crypt": "^2.5 || ^3.0",
```

Comment:
@Maks3w the Composer documentation for that is inconsistent, and uses both. As the parser accepts either, I think using `||` makes sense, for its affiliation with the language itself.